src/Components/Album.jsx

```diff
@@ -1,11 +1,37 @@
-//import React from 'react';
-//import { AlbumName } from "./AlbumName";
+import React from "react";
+import { Albums } from "../App";
```

Comment:
In the Album component, you should import all the album children Components:
AlbumName
ArtistName
Header
CoverImage

Album should not be imported from its parent component - the App component, but exported from this component, for example - export Album OR const Album and later export